pages/api/xAPI/agents/profile/index.ts

```diff
@@ -0,0 +1,87 @@
+import Cors from "cors";
+import deepEqual from "deep-equal";
+import type { NextRequest, NextResponse } from "next/server";
+import { apiHandler } from "../../../helpers/api/api-handler";
+import middleware, { Next } from "../../../helpers/api/request-sanitizers";
+import dbClient from "../../../../../lib/db";
```

Comment:
It doesn't seem like we are using `Cors`, `deepEqual`, `NextRequest`, `NextResponse`, `Next`,  or `dbClient`.